### ISSUE TITLE
Windows encodings

### ISSUE THREAD

**Issue by juliakreutzer** [joeynmt/joeynmt#152]
Mostly fixed in #150 but the following issues remain (by @ConstantineLignos) :

- A dependency is broken and to fix it you need to use Python 3.8 and pip install pywin32==225 to fix it. I
- Adding MacOS and Windows to Travis is going to be a fair amount of work since the standard Python Travis environment isn't available on either.
- MonoDataset can read from stdin, which isn't guaranteed to have a specific encoding. This is a little awkward to address and requires more testing, so I didn't address it for now.

**Comment by juliakreutzer** [comment]
v2.0 removes travis, potential input encoding issue is out of scope.

**Comment by may-** [comment]
Note: in windows, need to set encoding of the logger explicitly, it seems.
```python
# in make_logger()
fh = logging.FileHandler(log_file.as_posix(), encoding="utf-8")
```
-> will be merged v2.1
